creator-node/scripts/user_metadata_to_cnode_missing_files.js

```diff
@@ -0,0 +1,330 @@
+const axios = require('axios')
```

Comment:
This file would not be committed. It would be run locally against different services but then would never be called again